app/stacks/global/front-door/waf-policy.tf

```diff
@@ -148,6 +148,15 @@ resource "azurerm_frontdoor_firewall_policy" "default" {
       operator       = "Equals"
       selector       = "backOfficeProjectUpdateContent"
     }
+

```

Comment:
I don't think we need it at the default level, probably best to keep this small and just have it in the back office section below.

Reply:
Removed from default as requested